schemas/2016-05-01-preview/Microsoft.MachineLearning.json

```diff
@@ -4,6 +4,32 @@
   "title": "Microsoft.MachineLearning",
   "description": "Microsoft MachineLearning Resource Types",
   "resourceDefinitions": {
+    "commitmentPlans" : {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.MachineLearning/commitmentPlans"
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2016-05-01-preview"
+          ]
+        },
+        "sku": {
```

Comment:
We don't accept custom properties at top level. Please create a "properties" property and put "sku" there. I think our deployment service blocks this also. (if you write a template like this and deploy it, you should get an error.)